.travis.yml

```diff
@@ -7,7 +7,11 @@ jdk:
   - oraclejdk8
 rvm:
   - 2.4.2
+before_script:
+  - cp config/database.yml.travis config/database.yml
+  - psql -c 'create database travis_ci_test;' -U postgres;
```

Comment:
Can't we just say 'mahonia_test' in line 12 and get rid of line 11?

Reply:
No. It's also necessary to create a role that would allow rails to log in and create the database, e.g. https://github.com/curationexperts/mahonia/pull/132/files#diff-04c6e90faac2675aa89e2176d2eec7d8R27. Using `travis_ci_test` uses the role that travis supports out of the box.

I had a small amount of trouble setting up the `mahonia` role and decided it was better to just go with travis's approach. See also: https://docs.travis-ci.com/user/database-setup/#PostgreSQL